### ISSUE TITLE
Emergency Avatar Dequip

### ISSUE THREAD

**Issue by lxw404** [Yellow-Dog-Man/Resonite-Issues#2980]
### Is your feature request related to a problem? Please describe.

When new users set up their avatars for the first time, there is a decent chance that their viewpoint or lasers will not be set up correctly and they will be unable to interact to dequip their new avatar.

### Describe the solution you'd like

There should be a facet/gesture/keybind to dequip your avatar in an emergency which reverts you to the default avatar.  There are times when this is more desired than respawning due to the fact that a user might lose their work by doing that instead.

### Describe alternatives you've considered

The community can build a facet for this (and already have created a button item for this with a keybind), however an official solution would be preferable as well as a clear communication on how to use this as a feature during a tutorial or such.

### Additional Context

I've too often had to tell new users that they have to get out of VR and then explain desktop controls to fix things which is not an ideal first user experience.

### Requesters

LuxKitty

**Comment by shiftyscales** [comment]
This issue has some overlap with others such as #20.

> there is a decent chance that their viewpoint or lasers will not be set up correctly

This line suggests to me however, our efforts would probably be better spent improving the avatar creator itself if it's a frequent source of problems. Having an option to step out of your avatar / equip a default avatar on command could still be useful independently, e.g. not having to spawn a second copy of your avatar when working on it externally, but ultimately, as you put it here- that is just masking /compensating for poor UI/UX of the creator itself.

**Comment by lxw404** [comment]
Yep exactly my thoughts on this, it's a good indication that the avatar creator needs to have a much lower probability of causing those issues.  As you mention, having an option to step out of your avatar with a facet/gesture/keybind would definitely still be useful since currently the alternative is to navigate your inventory to find a temporary avatar which takes quite a bit longer.
